assets/js/main.js

```diff
@@ -69,3 +79,14 @@ $(".show-hide").click(function() {
     $(this).removeClass("ms-icon icon-hide").addClass("ms-icon icon-view");
   }
 });
+
+  // languague
+  // $('.language-selector .option').click(function(){
+  //   $(this).toggleClass('open');
+  // });
+
+  // $(window).click(function(e) {
+  //   if ($('.language-selector').has(e.target).length == 0 && !$('.language-selector').is(e.target)) {
+  //     $('.language-selector .option').removeClass('open');
+  //   }
+  // });
```

Comment:
Could you remove commented code?

Reply:
Sorry, I thought I had deleted the comments